meteor/server/collections/index.ts

```diff
@@ -0,0 +1,445 @@
+import { PieceInstance } from '@sofie-automation/corelib/dist/dataModel/PieceInstance'
+import { DBRundown } from '@sofie-automation/corelib/dist/dataModel/Rundown'
+import { RundownBaselineAdLibAction } from '@sofie-automation/corelib/dist/dataModel/RundownBaselineAdLibAction'
+import { RundownBaselineAdLibItem } from '@sofie-automation/corelib/dist/dataModel/RundownBaselineAdLibPiece'
+import { RundownBaselineObj } from '@sofie-automation/corelib/dist/dataModel/RundownBaselineObj'
+import { DBRundownPlaylist } from '@sofie-automation/corelib/dist/dataModel/RundownPlaylist'
+import { WorkerThreadStatus } from '@sofie-automation/corelib/dist/dataModel/WorkerThreads'
+import { MediaObject } from '@sofie-automation/shared-lib/dist/core/model/MediaObjects'
+import { MediaWorkFlow } from '@sofie-automation/shared-lib/dist/core/model/MediaWorkFlows'
+import { MediaWorkFlowStep } from '@sofie-automation/shared-lib/dist/core/model/MediaWorkFlowSteps'
+import { Meteor } from 'meteor/meteor'
+import { Bucket } from '../../lib/collections/Buckets'
+import { ICoreSystem } from '../../lib/collections/CoreSystem'
+import { Evaluation } from '../../lib/collections/Evaluations'
+import { ExpectedPackageDB } from '../../lib/collections/ExpectedPackages'
+import { ExpectedPackageWorkStatus } from '../../lib/collections/ExpectedPackageWorkStatuses'
+import { ExpectedPlayoutItem } from '../../lib/collections/ExpectedPlayoutItems'
+import { DBOrganization } from '../../lib/collections/Organization'
+import { PartInstance } from '../../lib/collections/PartInstances'
+import { Part } from '../../lib/collections/Parts'
+import { PeripheralDevice } from '../../lib/collections/PeripheralDevices'
+import { RundownLayoutBase } from '../../lib/collections/RundownLayouts'
+import { Segment } from '../../lib/collections/Segments'
+import { ShowStyleBase } from '../../lib/collections/ShowStyleBases'
+import { ShowStyleVariant } from '../../lib/collections/ShowStyleVariants'
+import { SnapshotItem } from '../../lib/collections/Snapshots'
+import { Studio } from '../../lib/collections/Studios'
+import { TimelineComplete } from '../../lib/collections/Timeline'
+import { TimelineDatastoreEntry } from '../../lib/collections/TimelineDatastore'
+import { TranslationsBundle } from '../../lib/collections/TranslationsBundles'
+import { DBTriggeredActions } from '../../lib/collections/TriggeredActions'
+import { UserActionsLogItem } from '../../lib/collections/UserActionsLog'
+import { DBUser } from '../../lib/collections/Users'
+import { WorkerStatus } from '../../lib/collections/Workers'
+import { registerIndex } from './indices'
+import { getCurrentTime } from '../../lib/lib'
+import { createAsyncMongoCollection, createAsyncOnlyMongoCollection, wrapMongoCollection } from './collection'
+import { ObserveChangesForHash, registerCollection } from './lib'
+
+export const AdLibActions = createAsyncMongoCollection<AdLibAction>(CollectionName.AdLibActions)
```

Comment:
I feel like this file should be split up into multiple, but im not sure how best to group everything as I think a file per collection will be too many